react/features/prejoin/reducer.js

```diff
@@ -64,9 +65,13 @@ ReducerRegistry.register(
         }
 
         case ADD_PREJOIN_VIDEO_TRACK: {
+            const videoTrack = action.value;
+            const videoTrackSet = videoTrack ? true : state.videoTrackSet;
+
             return {
                 ...state,
-                videoTrack: action.value
+                videoTrack,
+                videoTrackSet
```

Comment:
Please don't do this. Having 2 state variables that need to be in sync is a recipe for introducing bugs later. It's very confusing to have all these:

```
    videoTrack: null,
    videoTrackSet: false,
    videoDisabled: false,
    videoMuted: false
```

Reply:
Removed duplicate state variables.